pylibftdi/device.py

```diff
@@ -7,6 +7,7 @@
 pylibftdi: https://github.com/codedstructure/pylibftdi
 
 """
+from __future__ import annotations
```

Comment:
This requires Python3.7+, whereas the setup.py classifiers claims support for Python3.6. I should probably remove the more specific classifiers anyway, and since Python3.6 is EOL not too worried by this, doesn't need changing unless you want to remove the specific Python3.6 entry in setup.py

Reply:
Bumped the versions specified in the classifiers to support 3.7 to 3.11. Added the `python_requires=">=3.7"` line to be explicit.